README.md

```diff
@@ -64,7 +75,7 @@ how we're organizing branches in our repository in the [Contribution Guide](http
 
 ## Need help to build or hack? ##
 
-Say hello on our [Slack][4] channel: `#mobile`.
+Say hello on our [Slack](wordpress.slack.com) channel: `#mobile`.
```

Comment:
Oops this needs an https:// in front of it - but we should probably reference https://chat.wordpress.org for the instructions on how to join.